pegasus/sites.v3/code.org/public/private/privileges/index.haml

```diff
@@ -14,50 +14,39 @@
   messages = []
 
   def grant_permission_to_email(email, permission_name)
-    exit = false
-
-    if user = USERS.where(email:email).first
-      #user = USERS.where(email:email).first
+    user = USERS.where(email: email).first
+    if !user
+      return "#{email} is not a user"
```

Comment:
Nit (know this is just moved code): admin might think this means they should input a username instead. Could maybe be `No user with email #{email}`